code/modules/games/cards.dm

```diff
@@ -36,10 +36,17 @@
 					P.back_icon = card_decl["back_icon"]
 				if(!isnull(card_decl["desc"]))
 					P.desc = card_decl["desc"]
-				if(!isnull(card_decl["use_custom_front"]))
-					P.use_custom_front = card_decl["use_custom_front"]
-				if(!isnull(card_decl["use_custom_back"]))
-					P.use_custom_back = card_decl["use_custom_back"]
+
+				try
+					if(!isnull(card_decl["use_custom_front"]))
+						var/card_front = card_decl["use_custom_front"]
+						P.use_custom_front = fexists(card_front) && file(card_front)
+					if(!isnull(card_decl["use_custom_back"]))
+						var/card_back = card_decl["use_custom_back"]
+						P.use_custom_back = fexists(card_back) && file(card_back)
+				catch(var/exception/E)
+					to_world_log("Exception while loading card icons: [E]")
```

Comment:
Same here; can pull it out into a separate proc to make sure it doesn't halt execution.

Reply:
Done.